client/js/socket-events/more.js

```diff
@@ -16,6 +15,12 @@ socket.on("more", function(data) {
 	const scrollable = chan.closest(".chat");
 	const heightOld = chan.height();
 
+	// If there are no more messages to show, just hide the button and do nothing else
+	if (!data.messages.length) {
```

Comment:
Do we need to take into consideration when we get more messages and previous ones will be pushed off the client?